src/main/lrsql/init/oidc.clj

```diff
@@ -250,14 +256,17 @@
   "Given webserver and LRS configs, return a vector of interceptors to apply to
   Admin UI routes. If webserver oidc-client-id is not specified, returns an
   empty vector."
-  [{:keys [oidc-issuer
-           oidc-client-id] :as webserver-config}
+  [{:keys                           [oidc-issuer
+           oidc-client-id
+           oidc-enable-local-admin] :as webserver-config}
```

Comment:
Formatting boo-boo?

Reply:
Yeah, I use `clojure-align` on emacs and sometimes it gets a bit spicy, thanks

Reply:
https://github.com/yetanalytics/lrsql/pull/217/commits/8f9de18a3ee34d574f4c30c33dca01020d730e1e